packages/swirl-components/src/components/swirl-thumbnail/swirl-thumbnail.stories.ts

```diff
@@ -4,10 +4,20 @@ import Docs from "./swirl-thumbnail.mdx";
 export default {
   argTypes: {
     showRemoveButton: {
-      description: "Only displayed with size `xl` and format `square`.",
+      description:
+        "Only displayed with size `xl` or `2xl` and format `square`.",
```

Comment:
**suggestion:** Maybe we can add this information to `showEditButton` as description as well. Plus, we could mention that it is not displayed if `showRemoveButton === true && size === 'xl'`